cmd/nexd/main.go

```diff
@@ -5,24 +5,32 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
+	"runtime"
 	"sort"
 	"sync"
 	"syscall"
 
 	"github.com/nexodus-io/nexodus/internal/nexodus"
+	"github.com/nexodus-io/nexodus/internal/util"
 	"github.com/urfave/cli/v2"
 	"go.uber.org/zap"
 )
 
 const (
-	nexodusLogEnv = "NEXD_LOGLEVEL"
+	nexodusLogEnv    = "NEXD_LOGLEVEL"
+	Linux            = "linux"
+	wireguardOptions = "Wireguard Options"
+	agentOptions     = "Agent Options"
+	apiserverOptions = "ApiServer Options"
```

Comment:
Our document on naming conventions suggests we'd use "Nexodus Service Options" here.